### PR TITLE
Anatomy page

### PR DESCRIPTION
**Pull request by EmmaRenauld**
Adding an "Intro to famous bundles" page.

### CODE REVIEW

File: source/arriving/anatomy.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+(COPIER LE LIEN SUR BRAIN DATA. PAS DISPO LÀ...)
+
+A few famous bundles
+********************
+
+Here are pictures of reference bundles. Here, we took the bundles used as reference when using Recobundles. They are in MNI space. To see an example of bundles recovered in a real subject, see the :ref:`ref_recobundles` page. The bundles used in the pictures below are the ones that were used when processing the `Tractoinferno <https://www.nature.com/articles/s41597-022-01833-1>`_ database. You can also discover similar bundles associated with the phantom from the `ISMRM 2015 Tractography Challenge <https://tractometer.org/ismrm2015/home/>`_. The differences between these simulated bundles and real anatomical ones were discussed in `Renauld et al, 2023 <https://www.nature.com/articles/s41598-023-28560-w>`_.
+
+
+- AC:
```
Comment (ThoumyreStanislas) [inline source/arriving/anatomy.rst:22]: AC : Anterior Commissure
FAT : Frontal Aslant Tract
FPT : Frontopontine Tract
ICP : Inferior Cerebellar Peduncle
IFOF : Inferior Fronto-Occipital Fasciculus
ILF : Inferior Longitudinal Fasciculus
MCP : Middle Cerebellar Peduncle
MdLF : Middle Longitudinal Fascicle
PC : Posterior Commissure
POPT : Parieto-Occipital Pontine Tract
PyT : Pyramidal Tract
SCP : Superior Cerebellar Peduncle
SLF : Superior Longitudinal Fasciculus
UF : Uncinate Fasciculus

Comment (EmmaRenauld) [reply source/arriving/anatomy.rst]: Thanks!

**Comment by EmmaRenauld** [comment]
Here's what it looks like for the moment.

[Discover the brain anatomy — scil-documentation documentation.pdf](https://github.com/scilus/scil-documentation/files/13967592/Discover.the.brain.anatomy.scil-documentation.documentation.pdf)

@mdesco , I would have put more images from the links I point to, but I'm not sure that I can with the 'droit d'auteurs'. What to you think?

**Comment by CHrlS98** [comment]
@EmmaRenauld Should it go in the "Intro to" sections instead of "arriving"? We're think of renaming the intro sections to something like "Handbook/Quick reference guide of computer sciences" or maybe it could be "Handbook of neuroinformatics"?

**Comment by EmmaRenauld** [comment]
Sure. When I created this, "intro to" were more Francois's stuff. But tell me where to put it and how to name it!

**Comment by karanphil** [comment]
This is great! I would indeed move this to "Intro to" for the moment. We will move it later when we figure out what to do with that section.

**Review by CHrlS98 — CHANGES_REQUESTED** [review]
Based on the discussion --> Move to "intro to" section and keep same name for now !

**Comment by EmmaRenauld** [comment]
When I made the change, it felt weird because the title of the section is "Intro to computer science":
image

And this is not. So I create a new section: "Intro to medical imaging". Maybe at some point we will have other things in there. Max's introduction to dMRI for instance?

**Review by CHrlS98 — APPROVED** [review]
All these "Intro to" pages could simply be under a "Quick reference" umbrella or something... but for now this is good enough for me.

**Pull request merged**